### PR TITLE
feat(morty): Add Morty

### PR DESCRIPTION
**Pull request by Poltergeist**
Morty is the test runner which is using the generated AST to run tests. This is the first POC

### CODE REVIEW

File: morty/package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "dreamathon_2017",
+  "version": "1.0.0",
+  "description": "",
+  "main": "runner.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (paegun) [inline morty/package.json:7]: This message is confusing. It implies that the script "test" can actually succeed if the user specifies a test to run.

Given that `morty/runner.js` is a great script to demonstrate the functionality, I'd suggest adding "morty" as a script.

**Comment by paegun** [comment]
This looks + works great @Poltergeist . +1

If you like, address the npm scripts section comment.

**Review by paegun — APPROVED** [review]
This looks + works great @Poltergeist . +1

If you like, address the npm scripts section comment.

**Comment by orlandohohmeier** [comment]
Wonder if we should move morty into _his_ own repo?

**Comment by Poltergeist** [comment]
@orlandohohmeier we could do that but but we could also stick to the monorepo approach, but this is more of a general discussion we should have offline.

I will rename the `runner.js` to `morty.js` runner was more of a working title anyway.

**Pull request closed without merging**